src/js/guestModules/checkedForm.js

```diff
@@ -76,70 +77,76 @@ const CheckedForm = {
         $id('checkedForm-results-loading').style.display = 'block';
         $id('checkedForm-results-container').style.display = 'none';
 
-        // TODO: bazka
-        console.log(which);
-        const str = '{"title":"Nowy Form","questions":[{"number":1,"type":"O","language":"EN","content":"1. Open Question","numberOfAnswers":"|","answers":[],"userAnswer":"Form"},{"number":2,"type":"W","language":"EN","content":"2. Closed","numberOfAnswers":3,"answers":["1","2","3"],"userAnswer":"2"}, {"number":3,"type":"l","language":"EN","content":"1. Numerical Question","numberOfAnswers":"|","answers":[],"userAnswer":"13"}],"owner":"metin90.1998"}';
-        const form = JSON.parse(str);
-
-        $id('checkedForm-form-title').innerHTML = form.title;
-        $id('checkedForm-form-content').innerHTML = '';
-
-        for (const [it, question] of form.questions.entries()) {
-            if (question.type.toLowerCase() === 'o') {
-                const questionDOM = createOpenQuestion(question.number, question.content);
-
-                if (which.points[it])
-                    questionDOM.querySelector('p').style.color = '#2fa34e';
-                else
-                    questionDOM.querySelector('p').style.color = '#da2747';
-
-                const input = questionDOM.querySelectorAll('input')[0];
-                input.disabled = true;
-                input.value = question.userAnswer;
-
-                $id('checkedForm-form-content')
-                    .appendChild(questionDOM);
-            } else if (question.type.toLowerCase() === 'w') {
-                const questionDOM = createClosedQuestion(question.number,
-                    question.content,
-                    question.answers);
-
-                if (which.points[it])
-                    questionDOM.querySelector('p').style.color = '#2fa34e';
-                else
-                    questionDOM.querySelector('p').style.color = '#da2747';
-
-                const inputs = questionDOM.querySelectorAll('input');
-                const labels = questionDOM.querySelectorAll('span');
-
-                const limit = inputs.length;
-                for (let i = 0; i < limit; i++) {
-                    inputs[i].disabled = true;
-                    if (labels[i].innerHTML === question.userAnswer)
-                        inputs[i].checked = true;
+        getFilledFormFromDatabase().then(str => {
```

Comment:
Jaki jest sens pobierania wszystkich formularzy? Lepiej dodac argument, ktory z backendu pobierze odpowiedni form

Reply:
Problem jest taki który opisałem jak powyżej, nie ma połączenia pomiędzy filledForm a result, po iteracji należy dodać id który pozwoli pobrać beż problemu odpowiedni form, obecne rozwiązanie jest nie za dobre

Reply:
Ale przecież nazwa formularza jest unikalna, wiec po co nam id?

Reply:
Dokładnie :D